implementation/src/main/java/io/smallrye/mutiny/Uni.java

```diff
@@ -389,6 +393,25 @@ static UniCombine combine() {
      */
     Uni<T> cache();
 
+    // TODO
+    Uni<T> cacheUntil(BooleanSupplier invalidationGuard);
+
+    // TODO
+    default Uni<T> cacheFor(Duration duration) {
```

Comment:
I liked the cache group, maybe also name it `memorize` with the deprecation renaming act?
Could make things clear.

Reply:
That'd be another step ;-)